CMakeLists.txt

```diff
@@ -30,6 +30,7 @@ include(${CMAKE_SOURCE_DIR}/src/js/CMakeLists.txt)
 option(libuv_buildtests "" OFF)
 add_subdirectory(deps/libuv EXCLUDE_FROM_ALL)
 add_subdirectory(deps/quickjs EXCLUDE_FROM_ALL)
+add_subdirectory(deps/replxx EXCLUDE_FROM_ALL)
```

Comment:
Set these options please, so we only build and link with the static library: https://github.com/AmokHuginnsson/replxx/blob/master/CMakeLists.txt#L7 Also diasble the tests.

Reply:
Done!